product_docs/docs/epas/14/deployment_options/epas_inst_linux/installing_epas_using_edb_repository/ppc64le/epas_sles15_ppcle.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "Installing EDB Postgres Advanced Server on SLES 15 ppc64le"
+navTitle: "SLES 15"
+---
+
+There are two steps to completing an installation:
+
+-   Setting up the repository
+-   Installing the package
+
+For each step, you must be logged in as superuser.
+
+```shell
+# To log in as a superuser:
+sudo su -
+```
+
+Before setting up the repository, you need to register with EDB. To receive credentials for the EDB repository, visit: [Repository Access Request](https://www.enterprisedb.com/repository-access-request).
+
+## Setting up the repository
+
+```shell
+# Install the repository configuration and enter your EDB repository
+# credentials when prompted
+zypper addrepo https://zypp.enterprisedb.com/suse/edb-sles.repo
+
+# Install SUSEConnect to register the host with SUSE, allowing access to SUSE repositories
+zypper install SUSEConnect
+
+# Register the host with SUSE, allowing access to SUSE repositories
+# You can skip this step if already registered
+# Replace 'REGISTRATION_CODE' and 'EMAIL' with your SUSE registration information
+SUSEConnect -r 'REGISTRATION_CODE' -e 'EMAIL'
+
+# Activate the required SUSE modules on SUSE 15
+SUSEConnect -p PackageHub/15.3/ppc64le
+
+# Refresh the metadata
+zypper refresh
+```
+
+## Installing the package
```

Comment:
Similarly as to what we discussed for EPRS, users have option to install all of EPAS core components or a subset (the packages listed in the Available packages topic) including the other products. We should mention that and provide a link and maybe an example or two. We are also having the same problem with the formatting of the install command (the ticks are showing in the output). Let's make these changes for the SLES topics at least.

Reply:
Resolved the ticks display in output.